### PR TITLE
Use normative language for error code per #25

### PR DESCRIPTION
**Pull request by aj-stein**

### CODE REVIEW

**Review by JAG-UK — APPROVED** [review]
I had decided to make this change locally on my live branch too :-)

**Review by SteveLasker — APPROVED** [review]
LGTM

**Review by achamayou — APPROVED** [review]

**Comment by aj-stein** [comment]
I know the PR change was ostensibly made locally by @JAG-UK, but if possible if you _will_ use this I would like to hold off for now. I am sure we have plenty of time. Thanks for the consideration!

File: draft-ietf-scitt-scrapi.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ To facilitate automated response to errors, this document defines a set of stand
 
 Error responses SHOULD be sent with the `Content-Type: application/problem+json` HTTP header.
 
-As an example, submitting a Signed Statement with an unsupported signature algorithm would return a `400 Bad Request` status code and the following body:
+As an example, submitting a Signed Statement with an unsupported signature algorithm MUST return a `400 Bad Request` status code and the following body:
```
Comment (SteveLasker) [inline draft-ietf-scitt-scrapi.md:116]:
```suggestion
Examples of errors may include:
```

**Review by achamayou — APPROVED** [review]

**Comment by aj-stein** [comment]
Apologies for the delay @SteveLasker. I have moved it from draft status to ready to go.

**Pull request merged**